packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -147,19 +160,35 @@ export default class PrepaidCard {
       );
     }
 
+    let txnHash = gnosisResult.ethereumTx.txHash;
+
     if (typeof onTxnHash === 'function') {
-      await onTxnHash(gnosisResult.ethereumTx.txHash);
+      await onTxnHash(txnHash);
     }
 
-    return await waitUntilTransactionMined(this.layer2Web3, gnosisResult.ethereumTx.txHash);
+    return await waitUntilTransactionMined(this.layer2Web3, txnHash);
   }
 
+  async transfer(txnHash: string): Promise<TransactionReceipt>;
   async transfer(
     prepaidCardAddress: string,
     newOwner: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async transfer(
+    prepaidCardAddressOrTxnHash: string,
+    newOwner?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!newOwner) {
```

Comment:
maybe a better test is that `prepaidCardAddressOrTxnHash` actually looks like an address, that way you are asserting on the value you care about rather than making an assertion on a side effect. All addresses have 40 characters plus "0x" (so 42 characters in total). All txn hashes have 64 characters plus "0x" (so 66 chars in total).